### PR TITLE
login.html

### PR DESCRIPTION
**Pull request by AlecBrookens**

### CODE REVIEW

**Review by Randall-S4 — APPROVED** [review]
Looks good

**Review by JosiahDeYong — APPROVED** [review]
Looks good!

File: TAScheduler/Templates/login.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+</head>
+<body>
+    <form id="loginForm" action="/login" method="post">
+        <label for="username">Username:</label>
+        <input type="text" id="username" name="username" required>
+        <label for="password">Password:</label>
+        <input type="password" id="password" name="password" required>
+        <input type="submit" value="Login">
+    </form>
+    <!-- Add validation here -->
```
Comment (Kiran-Ee) [inline TAScheduler/Templates/login.html:15]: Missing "forgot password" feature - I know this wasn't on the old prototype, but I am going to update that today

File: TAScheduler/Templates/login.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Login</title>
+    <!-- Add CSS styling here -->
+</head>
+<body>
+    <form id="loginForm" action="/login" method="post">
```
Comment (Kiran-Ee) [inline TAScheduler/Templates/login.html:8]: Shouldn't forms w/ post methods need "{% csrf_token %}" ?

**Review by Kiran-Ee — COMMENTED** [review]
Review Kiran's Changes ...

**Pull request merged**